display.go

```diff
@@ -75,7 +78,103 @@ type certWithName struct {
 	cert *x509.Certificate
 }
 
-func displayCertFromX509(block *pem.Block) {
+type dn struct {
+	CommonName         string   `json:"common_name"`
+	Organization       []string `json:"organization"`
+	OrganizationalUnit []string `json:"organizational_unit"`
+	Country            []string `json:"country"`
+	Locality           []string `json:"locality"`
+	KeyId              string   `json:"key_id,omitempty"`
+}
+
+type basicConstraints struct {
+	IsCA       bool `json:"is_ca"`
+	MaxPathLen int  `json:"pathlen"`
+}
+
+type nameConstraints struct {
+	Critical            bool     `json:"critical"`
+	PermittedDNSDomains []string `json:"permitted_dns_domains"`
```

Comment:
Also `permitted_dns_domains` might be misleading. What about negative constraints?